dapr/clients/grpc/_helpers.py

```diff
@@ -79,11 +80,92 @@ def to_str(data: Union[str, bytes]) -> str:
         raise f'invalid data type {type(data)}'
 
 
+def parse_endpoint(address: str) -> Tuple[str, str, int]:
```

Comment:
Let's move this somewhere else please. How about we create a new `helpers.py` in `conf`, so this can be imported as `from dapr.conf.helpers import parse_endpoint`